activerecord/lib/active_record/middleware/database_selector.rb

```diff
@@ -43,7 +43,7 @@ def initialize(app, resolver_klass = Resolver, operations_klass = Resolver::Sess
 
       attr_reader :resolver_klass, :operations_klass
 
-      # Middleware that determines which database connection to use in a mutliple
+      # Middleware that determines which database connection to use in a multiple
```

Comment:
Can you keep just this and revert all other changes.

Reply:
sure